### PR TITLE
PR for Issue 488

### PR DESCRIPTION
**Pull request by brentzundel**
This PR add clarifying text and edits to the Presentation section in response to issue #488

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1639,19 +1650,25 @@ <h3>Presentations</h3>
...
+        <h4>Presentations using derived credentials</h4>
+
         <p>
 Some zero-knowledge cryptography schemes might enable <a>holders</a> to
-indirectly prove they hold a <a>credential</a> without revealing the
-<a>credential</a> itself. In these schemes, the original attribute, such as
-date of birth, might be translated into another value, such as "over the age
-of 15", which is cryptographically asserted such that a <a>verifier</a> can
-trust the value if they trust the <a>issuer</a>.
+indirectly prove they hold <a>caims</a> from a <a>verifiable credential</a>
```
[the remark this answers is not in the document]
Comment (brentzundel) [reply index.html]: Thank you @TallTed 
fixed in [ce01b83](https://github.com/w3c/vc-data-model/pull/538/commits/ce01b8328e0be6c46dbc54bf0e112d61bc8cf38c)

File: index.html [hunk 2]
```diff
@@ -1639,19 +1650,25 @@ <h3>Presentations</h3>
...
 above are <a>verifiable credentials</a>, as described by this specification. The
 contents of the <code>proof</code> <a>property</a> are proofs, as described by
-the Linked Data Proofs [[?LD-PROOFS]] specification. The <code>id</code>
-<a>property</a> is optional and MAY be used to provide a unique identifier for
-the presentation. The value associated with the <code>id</code> property MUST
-be a <a>URI</a>.
+the Linked Data Proofs [[?LD-PROOFS]] specification.
         </p>
 
+        <h4>Presentations using derived credentials</h4>
```
Comment (msporny) [inline index.html]: Capitalize -- "Presentations Using Derived Credentials"

File: index.html [hunk 3]
```diff
@@ -1639,19 +1650,25 @@ <h3>Presentations</h3>
...
         <p>
 Some zero-knowledge cryptography schemes might enable <a>holders</a> to
-indirectly prove they hold a <a>credential</a> without revealing the
-<a>credential</a> itself. In these schemes, the original attribute, such as
-date of birth, might be translated into another value, such as "over the age
-of 15", which is cryptographically asserted such that a <a>verifier</a> can
-trust the value if they trust the <a>issuer</a>.
+indirectly prove they hold <a>claims</a> from a <a>verifiable credential</a>
+without revealing the <a>verifiable credential</a> itself. In these schemes, a
+claim from a <a>verifiable credential</a> might be used to derived a presented
```
Comment (msporny) [inline index.html]: Fix `<a>claim</a>`.

Comment (msporny) [reply index.html]: Fix "derived" => "derive"

File: index.html [hunk 4]
```diff
@@ -1639,19 +1650,25 @@ <h3>Presentations</h3>
...
+value which is cryptographically asserted such that a <a>verifier</a> can trust
+the value if they trust the <a>issuer</a>.
+        </p>
+        <p>
+For example, a <a>verifiable credential</a> that contains the <a>claim</a>
+<code>date of birth</code> might be used to derived the presented value
```
Comment (msporny) [inline index.html]: Fix "derived" => "derive"

**Pull request merged**